commit f0861f82d9976fab8624f056fa6880e6f420e89f
tree 08227b2ab0d5df3770ae17d3d31eacdfe26f9867
parent 92548d601c45d76db337795d71c34846631dc7d6
author Masakazu Mokuno <mokuno@sm.sony.co.jp> 1184920382 +0900
committer Jeff Garzik <jeff@garzik.org> 1185308920 -0400

    ps3: use ethX as the name of irq
    
    Use net_device name for registration of irq as many network drivers do.
    
    Signed-off-by: Masakazu Mokuno <mokuno@sm.sony.co.jp>
    Signed-off-by: Jeff Garzik <jeff@garzik.org>

---
 drivers/net/gelic_net.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/gelic_net.c	2007-08-27 14:01:21.000000000 -0400
+++ b/drivers/net/gelic_net.c	2007-08-27 14:03:56.000000000 -0400
@@ -1075,7 +1075,7 @@ static int gelic_net_open_device(struct 
 	}
 
 	result = request_irq(card->netdev->irq, gelic_net_interrupt,
-			     IRQF_DISABLED, "gelic network", card->netdev);
+			     IRQF_DISABLED, card->netdev->name, card->netdev);
 
 	if (result) {
 		dev_info(ctodev(card), "%s:%d: request_irq failed (%d)\n",
